app/views/users/passwords/edit.html.erb

```diff
@@ -1,26 +1,40 @@
 <% self.title = t('titles.edit_info.password') %>
 
+<% if @update_password_presenter.required_password_change? %>
+  <%= render AlertComponent.new(
+        type: :warning,
+        class: 'margin-bottom-4',
+      ) do
+        t('users.password_compromised.warning', app_name: APP_NAME)
+      end
+  %>
+<% end %>
+
 <%= render PageHeadingComponent.new.with_content(t('headings.edit_info.password')) %>
 
-<p id="password-description">
-  <%= t('instructions.password.info.lead_html', min_length: Devise.password_length.first) %>
-</p>
+<% unless @update_password_presenter.required_password_change? %>
+  <p id="password-description">
```

Comment:
This element is referenced by `aria-describedby` for the field below, but it's conditionally rendered, which can mean it references an element that doesn't exist.

I'd suggest either conditionally assigning the `aria-describedby` based on `required_password_change?`, or always ensuring a `password-description` element exists with some description. We can use [`have_valid_idrefs`](https://github.com/18F/identity-idp/blob/9394620eeee7a3f6a6d03b4f475a329d83c1f499/spec/support/matchers/accessibility.rb#L1) (included in [`expect_page_to_have_no_accessibility_violations`](https://github.com/18F/identity-idp/blob/9394620eeee7a3f6a6d03b4f475a329d83c1f499/spec/support/matchers/accessibility.rb#L329C5-L329C52)) or [`have_description`](https://github.com/18F/identity-idp/blob/9394620eeee7a3f6a6d03b4f475a329d83c1f499/spec/support/matchers/accessibility.rb#L115) if we wanted to add some test assertions around the expected behavior.